### DISCUSSION TITLE
API to measure text width

### DISCUSSION THREAD

**Discussion by skysteve** [elgatosf/elgato#138] [Ideas]
Please expose the API used to calculate the width/clipping of text. I'm assuming this exists given the ability to specify [text-overflow](https://docs.elgato.com/streamdeck/sdk/references/touchscreen-layout/#text-text-overflow). It would be helpful to be able to know the width of text when rendered/where it would be cut such that the text could re-worded/split across multiple lines if appropriate.

**Comment by ChekTek** [comment]
Thanks for the suggestion! I will relay the feature request to the team. 👍
